examples/seismic/acoustic/operators.py

```diff
@@ -54,13 +75,20 @@ def iso_stencil(field, m, s, damp, kernel, **kwargs):
 def ForwardOperator(model, geometry, space_order=4,
                     save=False, kernel='OT2', **kwargs):
     """
-    Constructor method for the forward modelling operator in an acoustic media
-
-    :param model: :class:`Model` object containing the physical parameters
-    :param source: :class:`PointData` object containing the source geometry
-    :param receiver: :class:`PointData` object containing the acquisition geometry
-    :param space_order: Space discretization order
-    :param save: Saving flag, True saves all time steps, False only the three
+    Constructor method for the forward modelling operator in an acoustic media.
+
+    Parameters
+    ----------
+    model : `Model`
+        Object containing the physical parameters.
+    source : `PointData`
```

Comment:
``source/receiver`` do not seem to appear in the signature, while ``geometry`` does. I think the old docstring is obsolete. Can you update it @mloubout , or can you tell @georgebisbas how to update it?

Reply:
`kernel` is also missing

Reply:
The docstring is updated in `Wavesolver`, I remember doing it so should be just copy paste it for src/rec => geometry.

Reply:
@georgebisbas let us know if this is clear enough